candidate-standard/clause_3_references.adoc

```diff
@@ -15,5 +15,5 @@ The following normative documents contain provisions that, through reference in
 * [[rfc2413]] Weibel, S., Kunze, J., Lagoze, C., Wolf, M.: **IETF RFC 2413, Dublin Core Metadata for Resource Discovery**, https://tools.ietf.org/rfc/rfc2413.txt[https://tools.ietf.org/rfc/rfc2413.txt]
 * [[ogc06-103r4]] Herring, J.: **Simple Feature Access - Part 1: Common Architecture**, http://portal.opengeospatial.org/files/?artifact_id=25355[http://portal.opengeospatial.org/files/?artifact_id=25355]
 * [[ogc18-010r7]] Lott, R.: **Well-Known Text representation of Coordinate Reference Systems**, http://docs.opengeospatial.org/is/18-010r7/18-010r7.html[http://docs.opengeospatial.org/is/18-010r7/18-010r7.html]
-
+* [[ogc17-069r3]] Portele, C., Vretanos, P, Heazel, C.: **OGC API - Features - Part 1: Core**, http://www.opengis.net/doc/IS/ogcapi-features-1/1.0[http://www.opengis.net/doc/IS/ogcapi-features-1/1.0]
```

Comment:
I think we want this in normative? I reference it in the items query now.

Reply:
Agreed.